packages/libs/pactjs/src/index.ts

```diff
@@ -0,0 +1,28 @@
+import { Pact } from './pact';
```

Comment:
This is for testing purposes, right?

Reply:
Yes. The `index.ts` is an outline of what the interface could look like